brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1747 Fix: Undefined index for load-google-fonts-locally for new user

### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-1747

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Reset the site with the Astra theme activate.
- Refresh the admin. You will see the issue in debug log.
- Switch to our branch and try doing the same. The issue gets resolved.

### Checklist:
